### ISSUE TITLE
Missing header files in Xcode

### ISSUE THREAD

**Issue by dylan-infinitus** [BranchMetrics/cordova-ionic-phonegap-branch-deep-linking-attribution#718]
### Describe the bug

I have a Unity app I am building for iOS. 
I have imported the Branch Unity package. 

Building for iOS requires opening the Unity build output folder in Xcode and archiving the project to upload to Apple Connect.

--

When opening in Xcode, the project is missing multiple header files. 

This has been mentioned in a few other issues:

https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/issues/18
https://github.com/BranchMetrics/cordova-ionic-phonegap-branch-deep-linking-attribution/pull/670

However, I am still receiving the error from a Unity build output. 

Has the Unity package been updated with the supposed fixes?

### Steps to reproduce

1. Import Unity package to a Unity project
2. Build for iOS
3. Open in Xcode
4. Attempt to Archive the project

### Expected behavior

The project should be archived correctly and available to upload to Apple Connect.

### SDK Version

SDK-1523

### Make and Model

Xcode

### OS

Xcode

**Comment by echo-branch** [comment]
Issue is for Unity, there's a corresponding ticket on that repo.

**Comment by dylan-infinitus** [comment]
Yep, that was my bad. I should have closed this after creating the new one. Cheers!
